admin_guide/managing_projects.adoc

```diff
@@ -369,6 +369,77 @@ effect.
 # master-restart controllers
 ----
 
+[[limit-projects-per-service-account]]
+== Enabling and Limiting Self-Provisioned Projects Per Service Account
+
+By default, service accounts cannot create projects. However, administrators can
+enable this capability per service account, and the number of self-provisioned
+projects requested by any given service account can be limited with the
+`ProjectRequestLimit`
+xref:../architecture/additional_concepts/admission_controllers.adoc#architecture-additional-concepts-admission-controllers[admission control plug-in].
+
+[NOTE]
+====
+If service accounts are allowed to create projects, you cannot trust any labels
+placed on them because project editors can manipulate those labels.
+====
+
+. Create a service account in the desired project, if it is does not exist:
++
+----
+$ oc create sa <sa_name>
+----
+
+. As a user with `cluster-admin` privileges, add the `self-provisioner` cluster
+role to the desired service account:
++
+----
+$ oc adm policy \
+    add-cluster-role-to-user self-provisioner \
+    system:serviceaccount:<project>:<sa_name>
+----
+
+. Edit the master configuration file (`/etc/origin/master/master-config.yaml`)
```

Comment:
No parentheses. And that's the 4.x style for a file name, not the 3.x style.

> Avoid using parentheses in running text because text in parentheses is decreased in importance.